src/main/java/org/opensearch/security/dlic/rest/api/ssl/CertificateInfo.java

```diff
@@ -51,6 +76,10 @@ public CertificateInfo(final StreamInput in) throws IOException {
 
     @Override
     public void writeTo(final StreamOutput out) throws IOException {
+        out.writeOptionalString(format);
```

Comment:
Whenever reading and writing is modified, isn't it recommended to add to the end? There may be issues in a mixed cluster if a message is serialized from a newer node and communicates with an older node that doesn't understand the new values.

Reply:
True